packages/runner/src/lib/event-manager.js

```diff
@@ -278,6 +278,20 @@ const eventManager = {
       this._clearAllCookies()
       this._setUnload()
     })
+
+    top.addEventListener('message', (event) => {
+      switch (event.data) {
+        case 'app:cross:domain:window:load':
+          return Cypress.action('app:cross:domain:window:load')
+        case 'cross:domain:driver:ready':
+          this.crossDomainDriverWindow = event.source
+
+          return Cypress.action('runner:cross:domain:driver:ready')
+        default:
+          // eslint-disable-next-line no-console
+          console.log('Unknown postMessage:', event.data)
```

Comment:
the reason to not `throw` here is because the AUT could emit a postMessage to top, and we shouldn't fail the test in that case?

Reply:
This is another bare-minimum implementation to get the ball rolling. We're planning more work on this in the future to put more thought into it and flesh it out more.